README.md

```diff
@@ -9,18 +9,34 @@ Demo
 ----
 [View Demo](https://rawgithub.com/stewartlord/identicon.js/master/demo.html)
 
+Options
+----
+* **hash** - [Optional] A unicode string that will be used to generate the image. Defaults to a random hash based on the current time.
+* **options** - [Optional] An options object used to customize the generated image.
+    * **background** - The background color expressed as an rgba value array to use for the image background. For example, use [255,0,0,255] for red. Defaults to an opaque light gray [240,240,240,255].
+    * **margin** - The decimal fraction of the size to use for margin. For example, use 0.2 for a 20% margin. Defaults to 0.1 for a 10% margin.
+    * **size** - The size in pixels of the height and width of the generated (square) image. Defaults to 64 pixels.
+
 Usage
 -----
 ```js
+// set options
```

Comment:
Would you be opposed to a basic usage example with just hash and inline size {size: 420} -- followed by the more complete usage?